frontend/packages/console-app/src/actions/creators/pdb-factory.ts

```diff
@@ -69,7 +68,7 @@ const getPDBActions = (
 };
 
 export const usePDBActions = (kindObj: K8sKind, resource: K8sPodControllerKind) => {
-  const [namespace] = useActiveNamespace();
+  const watchNS = resource?.metadata?.namespace;
```

Comment:
nit, should we just keep the name "namespace"?
```suggestion
  const namespace = resource?.metadata?.namespace;
```

Reply:
@TheRealJon The lint wouldn't allow `resource?.metadata?.namespace` in dependency.

Reply:
Right, I'm saying to just name the const "namespace" instead of watchNS, which will make this PR a one-line change. But it's not really necessary, just a nit.